services/account/controllers/AccountController.ts

```diff
@@ -10,7 +11,30 @@ export class AccountController {
     this.repository = new AccountRepository();
   }
 
-  async getAccount(accountNumber) {
-    return await this.repository.find(accountNumber);
+  async getAccount(filter) {
+    try {
```

Comment:
I understand your intention. What I pointed out is that it's not necessary to try/catch and return a rejected promise. JavaScript will do it for you.